wgsl/index.bs

```diff
@@ -1314,7 +1314,7 @@ For example:
 
 <div class='example wgsl global-scope' heading='Mapping a texture_storage_ro_1d variable to SPIR-V'>
   <xmp>
-      var tbuf : texture_storage_ro_1d<rgba8unorm>;
+      var<uniform> tbuf : texture_storage_ro_1d<rgba8unorm>;
```

Comment:
textures don't need explicit storage qualifiers in WGSL

Reply:
Looks like we need to fix tint then!

`example-5c28da10.wgsl:1:5 error v-0022: global variables must have a storage class`

Reply:
Yea, Tint is lagging the spec change here.

Reply:
Force-pushed a revert of these two lines